app/server/app/public/data/config/services.json

```diff
@@ -45,6 +45,8 @@
     "queryStringFirstPart": "query?where=IS_DEFAULT%3D%27Y%27+and+STATE+in+%28",
     "queryStringSecondPart": "%29&outFields=*&returnGeometry=false&returnTrueCurves=false&returnIdsOnly=false&returnCountOnly=false&returnZ=false&returnM=false&returnDistinctValues=false&returnExtentsOnly=false&f=json"
   },
+  "wildScenicRivers": "https://services1.arcgis.com/fBc8EJBxQRMcHlei/arcgis/rest/services/WildScenicRiversV2_AGOL/FeatureServer/1/",
+  "protectedAreasDatabase": "https://gis1.usgs.gov/arcgis/rest/services/padus2/protection_mechanism_category/MapServer/0/query?f=json&where=OBJECTID=1&returnCountOnly=true",
```

Comment:
These two items need to be added to the `googleAnalyticsMapping` below. Also, should these be added to the `services-attains.json` file? I'm not sure if that one gets updated with each release or not. If it isn't updated then we can probably leave that json file as is.

Reply:
@cschwinderg good time to ask, is the services-attains.json file used for anything?

Reply:
I think it is just used for https://mywaterway-attains.app.cloud.gov/. 

@coobr01 Does the Attains version of HMW get updated when HMW is released to production?

Reply:
No, its a manual deployment. I can update it whenever you all want me to . But please update the attains.json file with the new services please. If you all merge the changes into ATTAINS branch, I'll manually deploy ATTAINS version.  We probably should change this if the ATTAINS instance is going to stay around I guess.